labs/src/main/scala/org/scalalabs/basic/lab03/RecursionPatternMatchingExercise.scala

```diff
@@ -37,7 +41,17 @@ object RecursionPatternMatchingExercise {
    * List(1,1,2,3,1,1) -> List(1,1), List(2), List(3), List(1,1)
    */
   def groupConsecutive[T](in: List[T]): List[List[T]] = {
-    error("fix me")
+    in match {
+      case Nil => Nil :: Nil
```

Comment:
Ah. Well, the solution has "case Nil => Nil," so I assume Scala would let you return either.